std/StandardToken.sol

```diff
@@ -3,20 +3,20 @@ pragma solidity ^0.4.0;
 import "./Token.sol";
 
 contract StandardToken is Token {
-	uint256 public totalSupply;
```

Comment:
@axic I've added a test `implement_interface_by_accessor` to test one of your requirements (what should work).  The other requirement (what should not work) is already checked in the test `function_clash_with_state_variable_accessor`.